.github/workflows/build.yml

```diff
@@ -376,3 +376,28 @@ jobs:
         with:
           name: 'cypress-visual-screenshots'
           path: '${{ github.workspace }}/apps/storybook/cypress-visual-screenshots'
+  a11y:
+    name: Test for a11y violations
+    needs: install
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Use Node 18.X
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18.x
+
+      - name: Cache node_modules
+        uses: actions/cache@v3
+        id: cache-node-modules
+        with:
+          path: |
+            node_modules
+            packages/*/node_modules
+            apps/*/node_modules
+            playgrounds/*/node_modules
+          key: modules-${{ runner.os }}-${{ hashFiles('yarn.lock') }}
+
+      - run: yarn test --filter=a11y
```

Comment:
Looks the problem was because of caching.
![](https://github.com/iTwin/iTwinUI/assets/9084735/d8538028-fcec-4072-9afb-fc2097af5bcb)

Try using this:
```suggestion
      - uses: cypress-io/github-action@v5
        with:
          build: yarn build --filter=itwinui-react
          working-directory: testing/a11y
```

Reply:
Committed the suggestion

Reply:
looks like that didn't work. Try this:

```
      - yarn build --filter=itwinui-react

      - uses: cypress-io/github-action@v5
        with:
          working-directory: testing/a11y
```

Reply:
done

Reply:
sorry, typo. the first line should be prefixed with `run: ` like this:
```yml
      - run: yarn build --filter=itwinui-react
```

Reply:
Fixed!

Reply:
Looks like it is correctly getting to that step now, but it is trying to run `e2e` tests instead of `component` tests.
![](https://github.com/iTwin/iTwinUI/assets/9084735/6f7c3247-0a15-4cb7-bc5e-d77d1ff3ae20)

Try adding `component: true` at the end
```
      - uses: cypress-io/github-action@v5
        with:
          working-directory: testing/a11y
          component: true
```

Reply:
Added

Reply:
Works correctly now. https://github.com/iTwin/iTwinUI/actions/runs/5412682827/jobs/9837126124

Just missing [one suggestion above](https://github.com/iTwin/iTwinUI/pull/1352#discussion_r1245535046) and then should be good to approve.